### PR TITLE
Adds translate as a property to element.animate

### PR DESCRIPTION
**Pull request by DivyanshiVashist**

### CODE REVIEW

File: test/translateTest.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/* global suite test assert */
+
+function checkTransformKeyframes (keyframes) {
```
Comment (courtneycb) [inline test/translateTest.js]: These functions are in their own file now 😎

Comment (DivyanshiVashist) [inline test/translateTest.js]: :+1:  :+1:

File: src/translate.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      if (unit !== 'px') {
+        // Element has invalid units for translation
+        return undefined;
+      }
+
+      valuesArray[i] = Number(numberString);
+    }
+
+    while (valuesArray.length < 3) {
+      valuesArray.push(0); // Add default value to unspecified axis
```
Comment (alancutter) [inline src/translate.js]: Add spec link that describes this behaviour.

File: test/toTransform.js [hunk 3]
```diff
@@ -1,30 +1,14 @@
...
-      assert.throws(function () {
-        toTransform({translate: '20px 30px 27px 90px'});
-      }, InvalidArgument);
-
-      assert.throws(function () {
-        toTransform({translate: 'garbage'});
-      }, InvalidArgument);
-
-      assert.throws(function () {
-        toTransform({translate: 'garbagepx'});
```
Comment (alancutter) [inline test/toTransform.js]: Don't remove test coverage for these inputs, just update expectations.

File: test/translateTest.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  test('translateTransform', function () {
+    isAnimationEqual({translate: ['none', '4px 5px 6px']}, {transform: ['translate3d(0px, 0px, 0px)', 'translate3d(4px, 5px, 6px)']});
+    isAnimationEqual({translate: ['1px', '2px']}, {transform: ['translate3d(1px, 0px, 0px)', 'translate3d(2px, 0px, 0px)']});
+    isAnimationEqual({translate: ['0.1px', '0.2px']}, {transform: ['translate3d(0.1px, 0px, 0px)', 'translate3d(0.2px, 0px, 0px)']});
+    isAnimationEqual({translate: ['1px 2px', '3px 4px']}, {transform: ['translate3d(1px, 2px, 0px)', 'translate3d(3px, 4px, 0px)']});
+    isAnimationEqual({translate: ['1px 2px 3px', '4px 5px 6px']}, {transform: ['translate3d(1px, 2px, 3px)', 'translate3d(4px, 5px, 6px)']});
+    // TODO: The test below should pass but the code doesn't like the exponential value
+    // isAnimationEqual({translate: ['1e3px 2px 3px', '4px 5px 6px']}, {transform: ['translate3d(1e3px, 2px, 3px)', 'translate3d(4px, 5px, 6px)']});
+    isAnimationEqual({translate: ['1px 2px', '2px']}, {transform: ['translate3d(1px, 2px, 0px)', 'translate3d(2px, 0px, 0px)']});
+  });
```
Comment (alancutter) [inline test/translateTest.js]: Add tests for invalid inputs just like scale and rotate has.

**Comment by alancutter** [comment]
The title/description should state what behaviour being changed.

**Pull request merged**